controllers/account/ec2.go

```diff
@@ -774,3 +792,62 @@ func cleanRegion(client awsclient.Client, logger logr.Logger, accountName string
 	}
 	return cleaned, nil
 }
+
+// Get the free instance type for the client's region
+func RetrieveAvailableMicroInstanceType(logger logr.Logger, awsClient awsclient.Client) (string, error) {
+	// FIXME: For unknown reasons attempting to use the free-tier-eligible
+	// filter from go returns *nothing*, but works fine from the CLI.
+	// HTTP-requests looks the same using both options.
+	availableTypes, err := awsClient.DescribeInstanceTypes(&ec2.DescribeInstanceTypesInput{
+		InstanceTypes: []*string{aws.String("t3.micro")},
```

Comment:
can we make the "t3.micro" string a named constant somewhere to describe what it is?